website/content/en/docs/best-practices/multi-tenancy.md

```diff
@@ -0,0 +1,192 @@
+linkTitle: "Multi-Tenancy"
+weight: 4
+description: This guide describes the best practices concepts to write Operators for Multi-Tenancy solutions.
+---
+
+## Operator Behavioral Guidelines
+
+### Managing Resources
+
+#### Resource Requests
+
+In Multi-tenant clusters, administrators must consider the effects that one tenant's workloads may have on others. 
+In an effort to prevent tenants from consuming all the resources on a cluster or blocking other workloads
+from being scheduled, many production clusters will define [ResourceQuota][k8s-resources-quotas] configurations.
+
+In an effort to support multi-tenant clusters with [ResourceQuotas][k8s-resource-quotas], operators should **always** 
+include reasonable memory and CPU resource requests for safe operations and avoid negatively impacting other workloads. 
+When integrating with [OLM][olm], resource requests can be defined by modifying the`config/manager/manager.yaml` file to 
+ensure that your CSV will be build with them when you run `make bundle` as shown below:
+
+```yml
+  ...
+  # TODO(user): Configure the resources accordingly based on the project requirements.
+  # More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
+  resources:
+    requests:
+      cpu: 10m
+      memory: 64Mi
+    ...
+```
+
+The [ResourceQuota][k8s-resources-quotas] configuration also applies to tenant workloads that are managed by 
+your Operator. Cluster administrators will typically set a [ResourceQuota][k8s-resources-quotas] for each 
+tenant's namespace as part of the onboarding. If a LimitRange with default values has not been created 
+in each namespace and your operator creates Pods/Deployments inside the tenant namespace which do not 
+specify at least resource requests for CPU and Memory, those Pods/Deployments may fail to 
+schedule and tenants may not be able to use your Operator. 
+
+Therefore, your operator is scheduled, it should **always** apply at least resource requests for `CPU` and
+memory for `Pods/Deployments` that it creates as part of the reconciliation logic. Ideally your 
+Operator also applies memory limits to those `Pods/Deployments`. You may also consider CPU limits.
+
+#### General guidelines
+
+Note that pretty much all controllers consume:
+
+- `CPU`: largely based on the number of reconciliations they perform, which are generally related to event activity for resources they're watching.
+- `Memory`: largely based on the number of primary resources that exist (multiplied by some factor based on the number of operand resources they need to watch as a result) via informer caches.
+
+Operator authors cannot anticipate how their operators will scale in all customer environments. A single configuration that fits all scenarios is not possible. Cluster administrators must be allowed to change the values in order to respond to their cluster's needs, such as resizing the control plane nodes to provide more resources.
```

Comment:
why cite resizing control plane nodes here? The thing operators need to do is ensure that the admin can change the resource requests/limits of the operator, and of the operands.